src/index.ts

```diff
@@ -183,6 +185,8 @@ export * from './operation-node/set-operation-node.js'
 export * from './operation-node/binary-operation-node.js'
 export * from './operation-node/unary-operation-node.js'
 export * from './operation-node/using-node.js'
+export * from './operation-node/json-path-leg-node.js'
+export * from './operation-node/json-path-node.js'
```

Comment:
Path reference node missing

Reply:
Nice catch! damn missed so many of these. 😕

Added @ af8aae26acbe0916b7d8c5715c3702cf29bc22e2.